Gordon360/Models/ViewModels/Model1.cs

```diff
@@ -0,0 +1,31 @@
+namespace Gordon360.Models.ViewModels
+{
+    using System;
+    using System.Data.Entity;
+    using System.Linq;
+
+    public class Model1 : DbContext
+    {
+        // Your context has been configured to use a 'Model1' connection string from your application's 
+        // configuration file (App.config or Web.config). By default, this connection string targets the 
+        // 'Gordon360.Models.ViewModels.Model1' database on your LocalDb instance. 
+        // 
+        // If you wish to target a different database and/or database provider, modify the 'Model1' 
+        // connection string in the application configuration file.
+        public Model1()
+            : base("name=Model1")
+        {
+        }
```

Comment:
It seems odd that this file is just now being added, but it doesn't seem to be causing issues.